star-dash/star-dash/Constants/PhysicsConstants.swift

```diff
@@ -56,7 +56,7 @@ struct PhysicsConstants {
         static let collectible = CollisionCategory.player
         static let obstacle = CollisionCategory.homingMissile
         static let powerUpBox = CollisionCategory.player
-        static let wall = CollisionCategory.player | CollisionCategory.homingMissile
```

Comment:
Was removing the homing missile category here accidental?

Reply:
No because we don't want to have missiles randomly disappearing when it hits an invisible wall